contributors/devel/sig-architecture/api_changes.md

```diff
@@ -1342,7 +1354,7 @@ The recommended place to do this is in the REST storage strategy's Validate/Vali
     }
     ```
 
-5. After at least one release, the feature can be promoted to beta or GA and enabled by default.
+5. After at least one release, the feature can be promoted to beta or GA. But it will only be enabled by default for GA.
```

Comment:
drop this change, this section of the doc is not referring to a new beta API version/endpoint

Reply:
Dropped.